src/java.desktop/windows/native/libjsound/PLATFORM_API_WinOS_MidiIn.cpp

```diff
@@ -70,8 +72,8 @@ void CALLBACK MIDI_IN_PutMessage( HMIDIIN hMidiIn, UINT wMsg, UINT_PTR dwInstanc
 
     case MIM_MOREDATA:
     case MIM_DATA:
-        TRACE3("  MIDI_IN_PutMessage: MIM_MOREDATA or MIM_DATA. status=%x  data1=%x  data2=%x\n",
-               dwParam1 & 0xFF, (dwParam1 & 0xFF00)>>8, (dwParam1 & 0xFF0000)>>16);
+        TRACE3("  MIDI_IN_PutMessage: MIM_MOREDATA or MIM_DATA. status=0x%" PRIxPTR " data1=0x%" PRIxPTR " data2=0x%" PRIxPTR "\n",
+               (uintptr_t)(dwParam1 & 0xFF), (uintptr_t)((dwParam1 & 0xFF00)>>8), (uintptr_t)((dwParam1 & 0xFF0000)>>16));
```

Comment:
Why cast to uintptr\_t? dwParam1's UINT\_PTR type is really just an unsigned int, so the existing %x seems correct